Documentation/network/concepts/ipam/multi-pool.rst

```diff
@@ -360,6 +360,30 @@ PodCIDRs allocated from ``CiliumPodIPPools`` can be announced to the network by
 the ``autoDirectNodeRoutes`` Helm option can be used to enable automatic routing
 between nodes on a L2 network.
 
+Masquerade Behaviour
+--------------------
+
+When combining multi-pool IPAM and BGP control plane, you may find it useful to not masquerade
+connections from such pools. As Pod IPs are advertised via BGP to your underlay network and
+return traffic can find its way back, it may not be desirable for the pod source IP to be
+masqueraded as the IP of the node it is on.
+
+It is not always possible to identify pods that should not be masqueraded with just destination
+IPs (via --ipvX-native-routing-cidr flag or ip-masq-agent rules) as there might be overlap
```

Comment:
Looks good thanks, I'd just mark these terms below as literals (I guess the spell checker may complain otherwise)

```suggestion
IPs (via ``--ipvX-native-routing-cidr`` flag or ``ip-masq-agent`` rules) as there might be overlap
```